src/bin/find.js

```diff
@@ -12,6 +12,7 @@ const options = {
   core: ['core'],
   verbose: ['verbose', 'v']
 };
+const optionsThatError = ['getUnusedRules', 'getDeprecatedRules'];
```

Comment:
I guess I’m confused by this line - why do we need this list? Any option could error and should exit nonzero in that case, no?

Reply:
If I understand correctly, the other options are informative only: show me my current rules, rules exposed by a plugin, etc.  In those cases, having rules to output isn't really an error.

The way the code was before I fixed this bug, only `getUnusedRules` would fail with an exit code of 1.

Reply:
Hmm. I guess I’d assume that each option should be able to determine its own success or failure state; but here it seems like the overarching runner is doing that?

In that case, maybe this is fine for now, but it should probably be rearchitechted later.

Reply:
I'd argue that there should be `process.exit(1)` on this line as well: https://github.com/sarbbottam/eslint-find-rules/blob/master/src/bin/find.js#L44

but I thought that might be out-of-scope for this PR.

Reply:
@ljharb Yeah, that does make more sense.  I think we'd have to change what the various rule finder methods return, which would be a breaking change if anyone is using the rule finder methods programmatically.

Reply:
Hey there,
is there anything left that stops this review from being approved? It looks good to me.

I totally agree that the determination about success or failure of the options' runners should be thought over and eventually reworked (as well as exiting with an exit code of 1 when no option has been provided), but I also agree that this is out of scope for this PR that should only fix the result of looking for deprecated rules.

Reply:
If you want to approve it, please do; I intentionally opened this as a non-blocking review :-)

Reply:
I didn't realize that 🙈